Applications/SlicerApp/qSlicerAppMainWindow.cxx

```diff
@@ -234,7 +234,7 @@ void qSlicerAppMainWindow::on_HelpInterfaceDocumentationAction_triggered()
 //---------------------------------------------------------------------------
 void qSlicerAppMainWindow::on_HelpSlicerPublicationsAction_triggered()
 {
-  QDesktopServices::openUrl(QUrl("http://www.spl.harvard.edu/publications/pages/display/?collection=11"));
+  QDesktopServices::openUrl(QUrl("https://scholar.google.com/scholar?&as_sdt=1%2C22&as_vis=1&q=%28%223D+Slicer%22+OR+%22slicer+org%22+OR+Slicer3D%29+-Slic3r+&btnG="));
```

Comment:
Shoud we instead open the community page ?

Reply:
This link https://www.slicer.org/wiki/Main_Page/SlicerCommunity#3D%20Slicer%20Enabled%20Research? 

That could be an option. The Slicer wiki feels a bit fragmented/dying at the moment. I'm not sure why that page about Slicer publications isn't just another Slicer.org page.

Reply:
Yes, probably that's cleaner.

Reply:
Good point.

Waiting the page is migrated to readthedocs or the new website, direct use of Google scholar makes sense.